packages/cardano-services/test/cli.test.ts

```diff
@@ -81,6 +81,19 @@ const assertMetaEndpoint = async (apiUrl: string, dataMatch: any) => {
   }
 };
 
+const assertStakePoolApyInResponse = async (apiUrl: string, assertFound: boolean) => {
+  expect.assertions(1);
+  await serverReady(apiUrl);
+  const headers = { 'Content-Type': 'application/json' };
+  const res = await axios.post(`${apiUrl}/stake-pool/search`, { headers, pagination: { limit: 1, startAt: 0 } });
+  const apy = res.data.pageResults[0].metrics.apy;
+  if (assertFound) {
+    expect(apy).not.toBeNaN();
```

Comment:
`expect({}).not.toBeNaN()` doesn't fail
```suggestion
    expect(typeof apy).toBe('number');
    expect(apy).not.toBeNaN();
```

Reply:
b441e8df7b84c5b91034dda0144a6d6e8a4370e2